src/main/sensors/barometer.c

```diff
@@ -336,8 +336,12 @@ static uint32_t recalculateBarometerTotal(uint8_t baroSampleCount, uint32_t pres
 }
 
 typedef enum {
-    BAROMETER_NEEDS_SAMPLES = 0,
-    BAROMETER_NEEDS_CALCULATION
+    BAROMETER_NEEDS_TEMP_READ = 0,
+    BAROMETER_NEEDS_TEMP_SAMPLE,
+    BAROMETER_NEEDS_PRES_START,
```

Comment:
could `PRES` be renamed `PRESSURE` here and in the 4 other cases?  Ideally `TEMP` should be `TEMPERATURE` too.  'temp' is often used as an abbreviation for temperature but it's also an abbreviation for 'temporary', I've never seen 'pres' used as an abbreviation for 'pressure'.  The goal is to make it readable, our IDEs/editors will save us from typing it.

Reply:
also, `PRES_START` looks like a typo for 'PRESS_START' and distracts the reader.

Reply:
Done. Thanks.